test.js

```diff
@@ -2,9 +2,17 @@
 'use strict';
 var assert = require('assert');
 var fs = require('fs');
+var util = require('util');
 var clearRequire = require('clear-require');
+var FixtureStdout = require('fixture-stdout');
+var stripAnsi = require('strip-ansi');
 var updateNotifier = require('./');
 
+function clearRequireAnd(modules, fn) {
+	modules.forEach(clearRequire);
+	fn();
```

Comment:
I don't know, seemed like a good idea at the time, but I guess it doesn't really help make things less verbose. 😅 I can change it.